cmd/deploy/proxy.go

```diff
@@ -255,6 +255,7 @@ func (ph *proxyHandler) getProxyHandler(token string, clusterConfig *rest.Config
 			r.Body = io.NopCloser(bytes.NewBuffer(b))
 		}
 
+		r.Host = destinationURL.Host
```

Comment:
Check line 242. We are proxying differently for PUT and POST so we might need to move this to line 231

Reply:
yes you are right, changed it